modules/ROOT/pages/migration-from-6x.adoc

```diff
@@ -3,62 +3,310 @@
+
+Options not available in **Advanced Template** plugin
+    
+* `template_cdate_classes`
+* `template_cdate_format`
+* `template_mdate_classes`
+* `template_mdate_format`
+* `template_replace_values`
+* `template_preview_replace_values`
+* `template_selected_content_classes`
 
-...
 
 [[ui-and-ux-elements-and-components]]
 == UI and UX elements and components
 
-...
+=== Improved
+
+[[ui-and-ux-elements-and-components-improved-how-styles-are-applied-when-resizing-rows-and-columns-for-tables]]
+==== Improved how styles are applied when resizing rows and columns for Tables.
+
+Previously added numerous style elements when resizing table rows and columns such as on the `table element`, `tr elements`, and `td elements`. This resulted in unappealing HTML output.
+
+{productname} 7.0 addresses this, now, the height input field has been moved from the `cell` properties dialog to the `row` properties dialog. Now, when an existing table is resized using the drag handles or the row properties dialog, inline css height declarations will be be stripped from cells and table elements and only applied on rows.
+
+[NOTE]
+{productname} 7.0 does not provide any fallback to revert to the old behavior.
+
+=== Fixed
+
+[[ui-and-ux-elements-and-components-fixing-the-table-style-by-css-option]]
+==== Fixing the `table_style_by_css` option
+
+Previously, tables were styled using CSS rather than attributes, aligning with modern web development practices.
+
+However, due to {productname}'s legacy predating this shift, a default option called `table_style_by_css` was introduced in {productname} 6 to accommodate this transition. Unfortunately, the implementation of this feature resulted in an unexpected behavior where the older border attribute was not updated alongside CSS changes, causing issues such as the disappearance of dashed outlines around tables with no borders.
+
+As a consequence, a breaking change has been initiated to address this issue, and the below changes are:
+
+* when a table's border `width` is set to `zero`, the **border attribute** will also be set to `zero`.
+
+=== Removed
+
+[[ui-and-ux-elements-and-components-force-notifications-to-have-a-close-button]]
+==== `Force` notifications to have a close button
+
+Currently, {productname} has the option to modify default notifications so that they do not have display a `X` (close button).
+
+{productname} is committed to enhance accessibility for all users, and aim to create consistency across all notifications and establish a standard tabbing flow for keyboard users.
+
+As of {productname} 7.0, the option `closeButton` has been removed and it will be defaulted to `true`.
+
+[IMPORTANT]
+If customers in-counter table styling issues, they will need to update their CSS to match the **new pattern** produced by {productname}.
```

Comment:
```suggestion
If customers encounter table styling issues, they will need to update their CSS to match the **new pattern** produced by {productname}.
```

Reply:
How does enforcing close button on notification dialog cause styling issues on tables?

Reply:
This seems to be in the incorrect section 🤔

Reply:
this was updated in latest commit